### DISCUSSION TITLE
Carelink

### DISCUSSION THREAD

**Discussion by FCdiab1** [NightscoutFoundation/xDrip#3424] [Q&A]
### Please don't open an issue if you haven't already consulted with the developers
- Please go back and use Discussions (on the previous page) to present your case if you haven't already.
- If you have consulted with the developers and were asked to open an issue, please proceed.
- Please also search the existing issues for similar problems before opening a new one.
- Please do not prefix you issue title with BUG: or LIBRE: for instance. The issue maintainer will tag every issue with appropriate labels.
- Add screenshots only if necessary, e.g. write the version number instead of adding a screenshot of the system status page.

### Subject of the issue
Link to the discussion thread:  
A clear and concise description of what the bug or request is.
Example: The BG graph shows negative data points between 3 and 5 am, which is incorrect and can only be an artefact.

### Your environment
- What version of xDrip, e.g. 2020.11.27 or 2011270800 or "Nightly build 27th Nov 2020". The version can be found at the system status page next to "Code:".
- What type of hardware data source and how do you get your data there (e.g. G6 or Libre with bluecon bridging device, Nightscout Follower, etc.)
- Is your issue specific to a device like and Android phone or a smartwatch? If yes, please provide the exact device name and type as well as the OS version (e.g. Samsung Galaxy S7 running Android 10.1).
- All settings related to the issue, e.g. activated auto calibration.

### Expected behavior
Example: BG values are always positive and should be displayed in line with the previous and following values, regardless of the time of day.

### Actual behavior
A clear and concise description of what actual happens.
Example: After enabling automatic calibration and entering a fingerstick test value, all new BG values received from the G6 sensor are displayed below the BG chart. Clicking on a data point shows a negative value.

### Steps to reproduce the behavior:
For example:
2. Enable setting '...'
3. Wait x minutes
4. Click on '...'

### Screenshots
If applicable, add screenshots to help explain your problem. Please do not add oversized screenshots!

**Comment by FCdiab1** [comment]
Last saturday Carelink out, after it Xdrip Carelink follower doesn't work

**Comment by Navid200** [comment]
Please post a screenshot of your classic status page: https://navid200.github.io/xDrip/docs/xDrip-Version.html

**Reply by eelcosimon** [reply]
Hi Navid
I can not make a screenshot.
Medtronic have updated their servers and server settings, roughly a week ago. After this, logging in via xDrip web integration, does not work anymore. You are kicked out immediately after logging in.
If you want to know anything in detail, I am more than willing to assist you.
Gr
Eelco

**Reply by Navid200** [reply]
There is not much anyone can do to help you remotely if you cannot show them a screenshot.
There are plenty of posts complaining about CareLink and most of them are because the user is using an old version of xDrip.

You can see in the release notes (https://navid200.github.io/xDrip/docs/ReleaseNotes.html) that there was an update related to CareLink on  December 5, 2023.  But, that is not our current stable release.  So, if you are still using the stable release, you need to update.

**Reply by eelcosimon** [reply]
Hi Navid, I do understand the situation and the 'unability' to help from a distance if you are not getting any intel of insights.
I always use the latest version of xDrip. Have added a printscreen to 'proof' this.
Furthermore, I have added the log files. At which you can see a code. 
And i have added the home screen, where you can see that xDrip is asking to log in. However, if i log in, then i am immediately logged out.
is this what you are looking?
![Screenshot_20240416_163306_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/63346792/30018da4-8126-441f-8c92-6e02f4b9a417)
![Screenshot_20240416_163333_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/63346792/0d56f2dc-ab9f-4fca-ac63-b1d2e754e281)
![Screenshot_20240416_163411_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/63346792/2fd774e1-1ca6-45cb-8d1f-ce1abb8bc33f)

**Reply by Navid200** [reply]
If you look at your status page, you can see the date is November 14, 2023.
As I explained before, the latest update to xDrip related to CareLink was added on December 5, 2023.
So, the version you are using, being the latest stable release, does not include the latest CareLink update.

I have explained how xDrip update works here: https://navid200.github.io/xDrip/docs/Updates.html

**Reply by eelcosimon** [reply]
Yes, Navid, That is exactly what I did. As you can see in the screenshot, xDrip clearly is stating that I am using the most recent version. If there is another way for me to get the December version, could you please send me the link?
![Screenshot_20240416_181255_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/63346792/8dec79da-1ef6-4779-98c8-65983b4f5ceb)
![Screenshot_20240416_181258_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/63346792/25117561-bf31-43a9-bdf3-964089d613f4)

**Reply by Navid200** [reply]
xDrip has different update channels.
By default, you are on the beta (stable) channel.

You can either change your update channel to alpha or you can install any release without changing your update channel.

I already gave you the link to release notes.  Here it is again: https://navid200.github.io/xDrip/docs/ReleaseNotes.html

If you look at that, you can see this:
![Screenshot 2024-04-16 121634](https://github.com/NightscoutFoundation/xDrip/assets/51497406/082d7c1c-9c61-4142-9c50-8201caa08f80)

You can click on the link in the left column to go to the specific release and install.

**Reply by Navid200** [reply]
And, no, you did not read the entire page because updating xDrip is not just one step.
Depending on what you are after, you may need to take actions other than just the very first step.

**Reply by eelcosimon** [reply]
ok, Navid.
